openfe/utils/integration_tools.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
We don't need 100+ lines of code to optionally import something

Reply:
Also integration_tools doesn't mean anything, optional_imports maybe?

Reply:
I think you could see it as tools to integrate other packages, but if you prefer optional_imports - no hard feelings. migth there be a case, where we want to use these tools for non optional stuff? Probably not.

Lines will be reduced after deciding for one solution :)